File: Maxglass.Ecommerce/Maxglass.Ecommerce.DataTransfer/Clientes/Responses/ClientePessoaJuridicaResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Maxglass.Ecommerce.DataTransfer.Clientes.Responses
{
    public class ClientePessoaJuridicaResponse : ClienteResponse
    {
        public string? Cnpj { get; set; }
        public string? RazaoSocial { get; set; }
        public string? InscricaoEstadual { get; set; }
        public string? NomeFantasia { get; set; }
    }
}

File: Maxglass.Ecommerce.Web/src/app/clientes/models/cliente-pessoa-juridica.response.ts
export class ClientePessoaJuridicaResponse {
  email: string;
  cep: string;
  cnpj: string;
  razaoSocial: string;
  inscricaoEstadual : string;
  nomeFantasia : string;


  constructor(params: Partial<ClientePessoaJuridicaResponse>) {
      this.email = params.email || null;
      this.cep = params.cep || null;
      this.cnpj = params.cnpj || null;
      this.razaoSocial = params.razaoSocial || null;
      this.inscricaoEstadual = params.inscricaoEstadual || null;
      this.nomeFantasia = params.nomeFantasia || null;
  }
}
